src/UserSettingsPage.cpp

```diff
@@ -750,6 +768,9 @@ UserSettings::save()
     settings.setValue(QStringLiteral("use_stun_server"), useStunServer_);
     settings.setValue(QStringLiteral("currentProfile"), profile_);
     settings.setValue(QStringLiteral("use_identicon"), useIdenticon_);
+#ifdef NHEKO_DBUS_SYS
```

Comment:
You don't need to ifdef this here, you can just move the enum value in the enum definition.

Reply:
Just move it to be the last enum value?

Reply:
Oh, I think I get it.

Reply:
In other words, if DBus is available, put the value before COUNT. Otherwise, put it after COUNT. Correct?